dt-notifications/notifications.php

```diff
@@ -432,10 +466,8 @@ public static function send_notification_on_channels( $user_id, $notification, $
         }
         if ( !in_array( 'email', $already_sent ) && dt_user_notification_is_enabled( $notification_type, 'email', $user_meta, $user_id ) ) {
             $user = get_userdata( $user_id );
-            if ( $user ){
```

Comment:
Why get rid of the user check. If a user is not found, the $user->user_email will throw an error.
@prykon